manifests/prometheus/list.yaml

```diff
@@ -0,0 +1,261 @@
+  kind: RoleBinding
+  metadata:
+    name: prometheus-telemeter-config
+    namespace: ${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: prometheus-telemeter-config
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: Role
+    name: prometheus-telemeter
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: prometheus-telemeter-config
+    namespace: ${NAMESPACE}
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    verbs:
+    - get
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - nodes
+    - services
+    - endpoints
+    - pods
+    verbs:
+    - get
+    - list
+    - watch
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+  spec:
+    port:
+      targetPort: https
+    tls:
+      termination: Reencrypt
+    to:
+      kind: Service
+      name: prometheus-telemeter
+- apiVersion: monitoring.coreos.com/v1
+  kind: PrometheusRule
+  metadata:
+    labels:
+      prometheus: telemeter
+      role: alert-rules
+    name: prometheus-telemeter-rules
+    namespace: ${NAMESPACE}
+  spec:
+    groups: []
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: prometheus-telemeter-tls
+    labels:
+      prometheus: telemeter
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+  spec:
+    ports:
+    - name: https
+      port: 9091
+      targetPort: https
+    selector:
+      app: prometheus
+      prometheus: telemeter
+    type: ClusterIP
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirectreference.prometheus-k8s: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"prometheus-telemeter"}}'
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+- apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
+  metadata:
+    labels:
+      k8s-app: prometheus
+    name: prometheus-telemeter
+    namespace: ${NAMESPACE}
+  spec:
+    endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      port: https
+      scheme: https
+      tlsConfig:
+        caFile: /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+        serverName: prometheus-telemeter.telemeter.svc
+    selector:
+      matchLabels:
+        prometheus: telemeter
+parameters:
```

Comment:
@jfchevrette the repo will now contain a Template manifest in `manifests/prometheus/list.yaml` that can be templated as needed and deployed.